### DISCUSSION TITLE
Add option to hide the Start Menu button

### DISCUSSION THREAD

**Discussion by BlueKiteCoder** [valinet/ExplorerPatcher#2135] [Ideas]
Suggestion: Cancel the windows logo button

In the taskbar at the bottom, I hope that the number of applications is 10, and the corresponding shortcut keys are (win 1 - win 0), which allows me to quickly open the application. As a result, the windows logo becomes redundant, and I use the "win" keyboard key to open the start menu every day (I use a pointing stick keyboard, so I don't use a mouse), and the windows logo icon makes me anxious. So I tried to use other software to hide it, but in the end all I could do was make it invisible, its position is still there, which makes the bottom 10 apps not centered (ugly), I hope your software can increase cancel The function of windows logo.

----------------------
![a](https://github.com/valinet/ExplorerPatcher/assets/45906849/5303a777-07a2-4bb5-b33c-0166fc986f26)
![x](https://github.com/valinet/ExplorerPatcher/assets/45906849/180f4384-9ffb-452a-918c-0b200258ffe0)

建议：取消windows logo按钮

在底部的任务栏中我希望应用数量是10个，对应快捷键是(win 1 - win 0),这使我能快速打开应用。于是，windows logo 变得多余了，日常我打开开始菜单用的是“win”键盘按键(我使用的是指点杆键盘，所以我不用鼠标)，windows logo 图标使我焦虑。于是我尝试使用其他软件对它进行隐藏，但最终做到的只有使它不可见，它的位置仍在那里，这让底部的10个应用无法居中(丑)，我希望你的软件可以增加取消windows logo的功能。

**Comment by pyrates999** [comment]
Please move this to discussions.

**Comment by ramojus** [comment]
For windows 10 style taskbar, startkiller worked for me.
